packages/hooks/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@aries-framework/hooks",
```

Comment:
Please copy the package.json (from e.g. redux-store) as closely as possible. It should at least contain the `publishConfig`, `scripts`, main and build files. (and probably some other properties)